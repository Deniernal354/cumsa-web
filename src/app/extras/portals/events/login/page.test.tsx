import { render } from "@testing-library/react";
import LoginPage from "./page";

test("LoginPage snapshot", () => {
  const { asFragment } = render(<LoginPage />);
  expect(asFragment()).toMatchSnapshot();
});
