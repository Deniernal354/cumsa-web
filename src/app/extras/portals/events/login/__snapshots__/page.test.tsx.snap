// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage snapshot 1`] = `
<DocumentFragment>
  <div
    class="min-h-[60vh] flex items-center justify-center"
  >
    <div
      class="w-full max-w-md"
    >
      <div
        class="text-center mb-8"
      >
        <div
          class="text-6xl mb-4"
        >
          üîê
        </div>
        <h1
          class="text-3xl font-extrabold text-[var(--blue)] mb-2"
        >
          Events Portal
        </h1>
        <p
          class="text-gray-600"
        >
          Authorized access only
        </p>
      </div>
      <form
        action="javascript:throw new Error('A React form was unexpectedly submitted. If you called form.submit() manually, consider using form.requestSubmit() instead. If you\\\\'re trying to use event.stopPropagation() in a submit event handler, consider also calling event.preventDefault().')"
        class="bg-white rounded-2xl p-8 shadow-lg border border-gray-100"
      >
        <div
          class="space-y-6"
        >
          <div>
            <label
              class="block text-sm font-semibold text-gray-700 mb-2"
            >
              Portal Password
            </label>
            <input
              class="w-full border-2 border-gray-200 rounded-lg px-4 py-3 focus:border-[var(--blue)] focus:outline-none transition-colors"
              name="password"
              placeholder="Enter access password"
              required=""
              type="password"
            />
            <p
              class="text-xs text-gray-500 mt-2"
            >
              Contact the admin if you need access credentials
            </p>
          </div>
          <button
            class="btn-cozy btn-primary w-full text-lg py-3"
            type="submit"
          >
            Access Portal
          </button>
        </div>
      </form>
      <div
        class="text-center mt-6"
      >
        <a
          class="text-sm text-[var(--blue)] hover:text-[var(--red)] underline"
          href="/events"
        >
          ‚Üê Back to Public Events
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
